packages/angular/test/apps/ng17/server.ts

```diff
@@ -0,0 +1,71 @@
+
+import 'zone.js/node';
+
+import { APP_BASE_HREF } from '@angular/common';
+import { CommonEngine } from '@angular/ssr';
```

Comment:
We need to overwrite this file to replace uses of `@nguniversal` with `@angular/ssr`